### DISCUSSION TITLE
Add delay to existing end point using proxy

### DISCUSSION THREAD

**Discussion by alehel** [mockoon/mockoon#408] [Help] [answered]
I'm experimenting with Mockoons proxy feature. I was wondering if there is a simple way to add a delay not only to mocked end points, but also to implemented end points. 

Let's say I have a spring boot application. It has the endpoint http://localhost:8080/employees which returns a list of employees. Can I use Mockoon proxy so that when the user accesses http://localhost:3000/employees, you first get a delay of x milliseconds, and then get the results from the actual API. Doing this for a mocked end point was simple enough, but can't figure out how to do this.

**Comment by 255kb** [comment, marked as answer]
The latency that you can set in Mockoon also applies to proxied endpoints: 
![image](https://user-images.githubusercontent.com/7489814/105630643-9ed0b180-5e4a-11eb-9663-e301c70faa05.png)
However, there is no currently no way to create complex scenarios, like only delaying the first call, randomize, etc.
Feel free to open an issue so we can track this.

**Reply by alehel** [reply]
I must be blind! I never noticed that there was a global latency option. Thanks.

**Reply by 255kb** [reply]
Glad it solves your problem! 😃
